fedimint-server/src/consensus/engine.rs

```diff
@@ -341,6 +352,21 @@ impl ConsensusEngine {
                     }
 
                     if let Some(UnitData::Batch(bytes)) = ordered_unit.data {
+                        if ordered_unit.creator == self.identity() {
+                             match timestamp_receiver.try_recv() {
+                                Ok((timestamp, bytes_len)) => {
+                                    if bytes.len() == bytes_len {
+                                        CONSENSUS_ORDERING_LATENCY_SECONDS.observe(timestamp.elapsed().as_secs_f64());
+                                    } else {
+                                        info!(target: LOG_CONSENSUS, "Not reporting ordering latency on possibly out of sync item");
+                                    }
```

Comment:
This will not synchronise after a restart, the offset will remain.

Reply:
Right. I need a loop. :D